### ISSUE TITLE
UI Suggestion. mouse zoom

### ISSUE THREAD

**Issue by newboerg** [fesch/Structorizer.Desktop#519]
How about a CTRL + Mousewheel zooming. Like its popular nowadays?

**Comment by GitMensch** [comment]
@newboerg Do you mean mouse scroll or mouse zoom?

**Comment by newboerg** [comment]
ment zoom, sorry. was in a hurry

**Comment by codemanyak** [comment]
@newboerg
Does your proposal refer to Arranger (where zooming has recently been implemented) or to the Structorizer work area (where zooming hasn't been planned by now)?

**Comment by newboerg** [comment]
well, the arranger would definitly gain from it as one has to push a key anyway.
and the work area has the change of fontsize which is quite like zooming.

**Comment by codemanyak** [comment]
I implemented it for Structorizer work area, Arranger, and output console. By now in the same sense as my browser, i.e. rotate forward to diminish / zoom out, rotate backward to enlarge / zoom in. I know some applications that do it the opposite way, however (which I ersonally find slightly more intuitive).
What do you suggest, in which rotation direction should it zom in? It's of course trivial to change it but I wouldn't like to change it AFTER the next version is published and user complained.

**Comment by GitMensch** [comment]
In my opinion rotating "up" should always zoom in (and that is what FF, Opera, NPP, VisualStudio, paint.net etc do). What application does it otherwise?

Question: Does anybody know what event is sent when a multi-touch-gesture for zoom-in is used?

**Comment by codemanyak** [comment]
@GitMensch 
> In my opinion rotating "up" should always zoom in

True. That was had been my intuition, too. But I was confused by the touch pad of my laptop because its gestures aren't always interpreted the same way as actual mouse wheel rotations would be. At least a recent driver update inverted the vertical scrolling direction. I had checked some applications on the PC with a mouse and others on the laptop with a touch pad and so I got the impression of different behaviour.

> What application does it otherwise?

Still, on searching different forums I had learned that there are many applications, which use a contrary zooming policy, e.g. QGIS versus ArcGIS, some CAD products differ as well etc.

So I will adhere to the already implemented and most usual way as default but make it simply configurable.
